app/settings.py

```diff
@@ -63,7 +63,11 @@ def get_env_or_fail(key):
 EQ_SERVER_SIDE_STORAGE_DATABASE_HOST = os.getenv('EQ_SERVER_SIDE_STORAGE_DATABASE_HOST')
 EQ_SERVER_SIDE_STORAGE_DATABASE_PORT = os.getenv('EQ_SERVER_SIDE_STORAGE_DATABASE_PORT', 5432)
 EQ_SERVER_SIDE_STORAGE_DATABASE_NAME = os.getenv('EQ_SERVER_SIDE_STORAGE_DATABASE_NAME', 'digitaleqrds')
-EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS = ensure_min(os.getenv('EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS', 10000), 1000)
+EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS = ensure_min(os.getenv('EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS', 10000),
+                                                       1000)
+
+EQ_PREPROD_ENV = os.getenv("https://surveypreprod.ons.gov.uk/")
+EQ_PROD_ENV = os.getenv("https://survey.onsdigital.gov.uk/")
```

Comment:
These need to be set to eq preprod and prod? not rrm ?

Reply:
This should be a variable of something like UPSTREAM_LOGIN_PAGE and not per environment